modules/persistent-storage-csi-vol-populator.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-vol-populator_{context}"]
+= Volume populators
+
+Volume populators use the `datasource` field in a persistent volume claim (PVC) spec to allow you to create pre-populated volumes.
+
+Volume population is currently enabled, and supported as a Technology Preview feature. However, {product-title} does not ship with any volume populators.
+
+:FeatureName: Volume populators
```

Comment:
To avoid "populators is" in the rendered output, would this be technically accurate if it were "Volume population" instead?